### ISSUE TITLE
React projects/templates load module dependencies instead of core/lob

### ISSUE THREAD

**Issue by bazal4o** [IgniteUI/igniteui-cli#193]
As a developer I need to be able to provide component dependencies for my template and only the required files should be loaded instead of core/lob, or dv
Webpack configuration

**Comment by damyanpetev** [comment]
**TL;DR:** We should probably postpone this and #194 for future updates in the Ignite UI for JS packages.

I've made a [test project repo](https://github.com/damyanpetev/cli-react-webpack-test/commits/master) to test out loading modules. Besides the base util requiring some [shimming](https://webpack.js.org/guides/shimming/), the standalone module files actually list just `"jquery-ui"` as AMD dependency which is more geared towards bower+require.js setup and doesn't play well in a webpack scenario. As the npm jQuery UI package doesn't have a single bundle exported that single dependency translates to just `"jquery-ui/ui/widget"`, requiring adding specific other modules depending on the Ignite UI files being loaded which in a way defeats the purpose of loading just modules.

The core and LoB bundle files actually do list `"jquery-ui"` and `"jquery-ui/ui/*"` modules dependencies and that's why they've been loading fine in our projects so far and I'm think we should stick with those for now.

I could add [util module shims](https://github.com/damyanpetev/cli-react-webpack-test/commit/40683a69b602e2675e927ad72c04aed828b1b912) to the project (even as commented out instructions) in case someone wants to give per-module loading a try in the meantime?

// CC @kdinev

**Comment by kdinev** [comment]
@damyanpetev Maybe we can add two separate `define`s for bower and npm?
